packages/data-provider/package.json

```diff
@@ -1,26 +1,40 @@
 {
-	"name": "data-provider",
-	"version": "1.0.0",
-	"description": "",
+	"name": "@overture-stack/lyric",
+	"version": "0.1.0",
+	"description": "Data Submission system",
 	"main": "dist/index.js",
 	"engines": {
 		"node": ">=20.0.0"
 	},
 	"scripts": {
 		"\n===============  Build  ===============": "",
-		"build:all": "rm -rf dist && tsc",
+		"build:all": "rimraf dist && tsc",
 		"\n===========  Dev environment  =========": "",
 		"start:dev": "nodemon"
 	},
-	"keywords": [],
+	"keywords": [
+		"overture",
+		"lyric"
+	],
 	"author": "Ontario Institute for Cancer Research",
-	"license": "ISC",
+	"files": [
+		"dist/"
+	],
+	"license": "AGPL-3.0-or-later",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/overture-stack/lyric.git"
+	},
+	"bugs": {
+		"url": "https://github.com/overture-stack/lyric/issues"
+	},
+	"homepage": "https://github.com/overture-stack/lyric#readme",
```

Comment:
Let's use the README files from each package to be their homepage instead of re-uising the repo's main README.

The main README should be about lyric as a product and the codebase itself, how to interact with the monorepo etc.

The README for each package should describe that package and how to use it (ie. how to write code with it, integrate it into other apps, important api features, dependencies, etc.).

Reply:
The `bugs` property is at repository level and `homepage` refers usually to a static website of the main project, this is used as an external link on the npm search.